src/serlo-editor/plugins/rows/editor-renderer.tsx

```diff
@@ -54,10 +55,16 @@ export function EditorRowRenderer({
         id: row.id,
         serialized: selectSerializedDocument(store.getState(), row.id),
         onDrop() {
+          // Remove the dragged plugin from its original rows plugin
           rows.set((list) => {
             const index = list.findIndex((id) => id === row.id)
             return R.remove(index, 1, list)
           })
+
+          // If the dragged plugin was the only plugin in the current rows plugin,
+          // add an empty text plugin to replace it
+          if (rows.length > 1) return
+          rows.insert(0, { plugin: EditorPluginType.Text })
```

Comment:
opinion: this is one of the cases where the early return does not help me understand.